components/Navbar.tsx

```diff
@@ -1,11 +1,22 @@
-import { AppBar, Toolbar, Typography } from "@mui/material";
+import { AppBar, Button, Tab, Tabs, Toolbar, Typography } from "@mui/material";
+import ShoppingCartIcon from '@mui/icons-material/ShoppingCart';
+import { useState } from "react";
 
 const Navbar = () => {
+  const [value,setValue]=useState(0)
```

Comment:
what is the use of state here?
were are you using the state value

Reply:
I am using state for the changing tab value in 
please check the 12 line